src/utils/colors.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
why not check this file? Is used a lot

Reply:
Honestly - the `pSBC` function grabbed from https://github.com/PimpTrizkit/PJs/wiki/12.-Shade,-Blend-and-Convert-a-Web-Color-(pSBC.js) looked really unpleasant to annotate, and typescript can't ignore blocks of code last i checked, just entire files.

Reply:
ack and understood. I'd consider moving to its own file since we use the other functions in a few places. Agreed w/ pSBC being a bear: I copied it uncritically from SO